### PR TITLE
Disable opening new positions

### PR DESCRIPTION
**Pull request by JaniAnttonen**

### CODE REVIEW

File: components/Trade/Positions/AvailableTokens.tsx [hunk 1]
```diff
@@ -103,7 +102,7 @@ export default function AvailableTokens({
 }
 
 function getColumns(
-  onBuyClick: HandleBuyClick,
+  _onBuyClick: HandleBuyClick,
```
Comment (vvp) [inline components/Trade/Positions/AvailableTokens.tsx:105]: Is the added underscore for keeping linter happy or ..? :slightly_smiling_face:

**Review by vvp — APPROVED** [review]
Looks good :+1:

**Comment by vvp** [comment]
Tried the deployment, but can't see any tokens so cannot verify if the change works - there's some errors in console, any idea if they're related?
  
image

File: components/Trade/Positions/AvailableTokens.tsx (hunk 1 above)
Comment (JaniAnttonen) [reply components/Trade/Positions/AvailableTokens.tsx:105]: Yes. :D

**Comment by JaniAnttonen** [comment]
> Tried the deployment, but can't see any tokens so cannot verify if the change works - there's some errors in console, any idea if they're related?
> 
> image

No idea anymore what these are, I assume they're empty queries that are tried with undefined inputs or something... Still have a test position and checked that the pricing and all seem to work. Merging.

**Pull request merged**